bucket/hokus.json

```
@@ -0,0 +1,35 @@
+{
+    "version": "0.6.1",
+    "description": "A modern CMS for Hugo",
+    "homepage": "https://www.hokuscms.com",
+    "license": "MIT",
+    "architecture": {
+        "64bit": {
+            "url": "https://github.com/julianoappelklein/hokus/releases/download/0.6.1/hokus-cms_win.exe#/dl.7z",
+            "hash": "e1e6ce154619b004b7b76fae68be92dda57b5223a54ad3a66545ed6f3c92d2ff",
+            "installer": {
+                "script": [
+                    "Expand-7zipArchive \"$dir\\`$PLUGINSDIR\\app-64.7z\" \"$dir\"",
+                    "Remove-Item \"$dir\\`$PLUGINSDIR\", \"$dir\\Uninstall*\" -Force -Recurse"
+                ]
+            }
+        }
+    },
+    "shortcuts": [
+        [
+            "Hokus CMS.exe",
+            "Hokus CMS"
+        ]
+    ],
+    "checkver": {
+        "url": "https://github.com/julianoappelklein/hokus/releases",
+        "regex": "tree/([\\d.]+)"
```

You can go with classic github

```suggestion
        "github": "https://github.com/julianoappelklein/hokus"
```

---

There isn't stable (or Latest) release now, so `github` doesn't work.

---

It will be redirected.